presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/StandardColumnMappings.java

```diff
@@ -415,12 +419,15 @@ public static LocalDateTime fromPrestoTimestamp(long value)
 
             case Types.NUMERIC:
             case Types.DECIMAL:
-                int decimalDigits = type.getDecimalDigits();
+                if (!type.getDecimalDigits().isPresent()) {
+                    return Optional.of(decimalColumnMapping(createDecimalType(Decimals.MAX_PRECISION, getDecimalDefaultScale(session)), getDecimalRoundingMode(session)));
```

Comment:
@kokosing @findepi 
By adding a configuration toggle like `decimal-rounding-enable`, it will be possible to return `Optional.empty()` when it's disabled and users will be able to fetch values as VARCHAR in case of overflow. WDYT ?